src/routes/MyProfile.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './MyProfile.css';
+
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const rocketsKey = Object.keys(rockets);
+  const reservedRocket = [];
+  rocketsKey.forEach((rocketId) => {
+    if (rockets[rocketId].reserved) {
+      reservedRocket.push({
+        id: rocketId,
+        name: rockets[rocketId].name,
+      });
+    }
+  });
+  const missions = useSelector((state) => state.missions.missions);
+  const missionJoined = missions.filter((mission) => {
+    if (mission.member !== true) {
+      return null;
+    }
+    return { mission };
+  });
+
+  return (
+    <div className="profile-display">
+      <div className="my-mission">
+        <h2>My Missions</h2>
+        <ul className="view-missions">
+          {missionJoined.length ? (
+            missionJoined.map((missions) => (
```

Comment:
- [ ] Greate job rendering a list of all joined missions, however, you can consider using filter() instead of the map() method, kindly try to implement this functionality it's a requirement

![Screenshot_20220722_213403](https://user-images.githubusercontent.com/25227022/180559701-5c477181-23ba-48ea-aa46-bb87e09c23f5.png)

Reply:
@Ismailco Kindly implement the `filter` in line https://github.com/darikmohammed/Space_Travelers_Hub/blob/28025b20390e1b175c0f1bde89304ffa62ff281a/src/routes/MyProfile.js#L18 
to filter all the joined missions.